### PR TITLE
Add simple dashboard

### LINKED ISSUE #18: Provide landing page & interface to control devices using preflashed images
Right now, users don't have a way to remotely manage the device or even power cycle it. 
Add an interface to manage device

### PR DESCRIPTION
**Pull request by iamsolankiamit**

### CODE REVIEW

File: ui/dashboard/server.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		"ACCOUNT_KEY",
+		req.body.accountKey
+	);
+	res.sendStatus(200);
+});
+
+// Shutdown handler
+app.post("/manage/shutdown", async (req, res) => {
+	res.sendStatus(200);
+	await axios.post(
```
Comment (sradevski) [inline ui/dashboard/server.js:50]: Not that it matters much, but we can also use `sdk.request.send` and remove `axios`.

File: ui/dashboard/views/index.ejs [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+					shutdownButton.onclick = function (event) {
+						fetch("/manage/shutdown", { method: "post" });
+						shutdownLoader.style.display = "block";
+					};
+
+					rebootButton.onclick = function (event) {
+						fetch("/manage/reboot", { method: "post" });
+						rebootLoader.style.display = "block";
+
+						window.setTimeout(function () {
```
Comment (sradevski) [inline ui/dashboard/views/index.ejs:180]: Why do we have this for reboot, but not shutdown?

**Review by sradevski — COMMENTED** [review]
I have some minor comments here and there, but considering the scope of the UI I am fine as it is, as long as it works :) About the design though, why should the buttons be so far apart? I just found it a bit strange it's all, but it's not a big deal.

File: ui/dashboard/views/index.ejs [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+							rebootLoader.style.display = "none";
+						}, 1000 * 60);
+					};
+
+					accountKeyButton.onclick = function (event) {
+						accountKeyModal.style.display = "block";
+					};
+
+					accountKeyForm.onsubmit = function (event) {
```
Comment (rahul-thakoor) [inline ui/dashboard/views/index.ejs:203]: the call to the endpoint is failing. `req.body` is `undefined`

Comment (iamsolankiamit) [reply ui/dashboard/views/index.ejs:203]: nice catch. Checking in a bit

File: ui/dashboard/server.js (hunk 1 above)
Comment (rahul-thakoor) [reply ui/dashboard/server.js:50]: TIL! thanks @sradevski !

**Comment by chrisys** [comment]
Visually and operationally this looks great to me, I defer to @rahul-thakoor and @sradevski to confirm their comments are addressed and then LGTM!

**Review by chrisys — APPROVED** [review]

**Comment by rahul-thakoor** [comment]
Looks good to me too @chrisys  . 
As far as changing the account goes, we run into issue #49 

this will addressed separately, as it is not related to the functionality of the dashboard itself.

thanks @iamsolankiamit

**Comment by rahul-thakoor** [comment]
@iamsolankiamit @chrisys I think we also discussed an info button for linking to what account key is?

File: ui/dashboard/views/index.ejs (hunk 2 above)
Comment (iamsolankiamit) [reply ui/dashboard/views/index.ejs:180]: Not Sure, @chrisys wanted the loader only for 60 sec for reboot and not for shutdown

**Comment by iamsolankiamit** [comment]
@rahul-thakoor yes we did, but we didn't finalise on the content/link

**Comment by rahul-thakoor** [comment]
I guess this? https://boinc.bakerlab.org/rosetta/weak_auth.php

**Comment by iamsolankiamit** [comment]
it asks you to login, do we have any link which is open and user can read it?

**Comment by rahul-thakoor** [comment]
https://boinc.berkeley.edu/wiki/Weak_account_key

**Comment by iamsolankiamit** [comment]
I'll take this up later today maybe in 5-6 hours. So if you want you can merge what we have now and i'll create another pr with that change.

File: ui/dashboard/views/index.ejs (hunk 2 above)
Comment (chrisys) [reply ui/dashboard/views/index.ejs:180]: My thinking was if the device has shutdown it's not coming back so we didn't need to time it out.

Comment (sradevski) [reply ui/dashboard/views/index.ejs:180]: We still show a spinner in the dashboard, and I think it still responds before it shuts down (once all processes are killed, etc.)? I am fine either way, just curious.

**Review by sradevski — APPROVED** [review]

**Review by rahul-thakoor — APPROVED** [review]

**Comment by chrisys** [comment]
@balena-ci retest

**Pull request merged**